src/obis-observer/commands/uplink/RemoveMeterResponse.ts

```diff
@@ -0,0 +1,89 @@
+import Command, {TCommandExampleList} from '../../Command.js';
+import CommandBinaryBuffer, {REQUEST_ID_SIZE, ICommandParameters} from '../../CommandBinaryBuffer.js';
+import {UPLINK} from '../../constants/directions.js';
+import {resultCodes} from '../../constants/index.js';
+
+
+/**
+ * IRemoveMeterResponseParameters command parameters
+ */
+interface IRemoveMeterResponseParameters extends ICommandParameters {
+    resultCode: number
+}
+
+
+const COMMAND_ID = 0x73;
+const COMMAND_SIZE = 1 + REQUEST_ID_SIZE;
```

Comment:
```suggestion
const COMMAND_SIZE = REQUEST_ID_SIZE + 1;
```

?

just to match the order
(and in other places if you wish)

Reply:
did it